src/libponyrt/lang/lsda.c

```diff
@@ -216,7 +216,9 @@ static bool lsda_init(lsda_t* lsda, exception_context_t* context)
   return true;
 }
 
-bool ponyint_lsda_scan(exception_context_t* context, uintptr_t* lp)
+bool ponyint_lsda_scan(uintptr_t* ttype_index __attribute__ ((unused)),
```

Comment:
the unused is required to keep gcc from complaining.

gcc has a warning for parameters that aren't used but are set. `ttype_index` is an output parameter so it needs to be set.